Update: Rewrite AI Gateway Getting Started guide (#8056)
Kong/docs.konghq.com: app/_src/gateway/ai-gateway/index.md

For more information, see [AI Metrics](/gateway/{{ page.release }}/production/monitoring/ai-metrics).

{% endif_version %}

> ## Quickstart script
> 
> Kong provides an interactive AI quickstart script, which launches a demo instance of {{site.base_gateway}} running AI Proxy.
> This script deploys {{site.base_gateway}} in a local Docker container in traditional mode, or as a data plane instance for {{site.konnect_short_name}}.
> 
> This option is for demo purposes only, and is not meant for production use.
> 
> {:.note}
> > **Note:**
> > Running this script prompts you for AI Provider API Keys which are used to configure authentication with
> > hosted AI providers. These keys are only passed to the {{site.base_gateway}} Docker container and are 
> > not otherwise transmitted outside the host machine.
> 

Reviewer: ```suggestion
Kong offers an interactive AI quickstart script that launches a demo instance of {{site.base_gateway}} running AI Proxy. 

```sh
curl -Ls https://get.konghq.com/ai | bash
```

The script will run in either traditional mode or as a data plane instance for {{site.konnect_short_name}}. You will be prompted to input an API key to configure authentication with an AI provider. This key will not be exposed outside of the host machine.
```

Reviewer: This rendered poorly, let me try again: 

This is what I mean: 

Kong offers an interactive AI quickstart script that launches a demo instance of {{site.base_gateway}} running AI Proxy. 

```sh
curl -Ls https://get.konghq.com/ai | bash
```

The script will run in either traditional mode or as a data plane instance for {{site.konnect_short_name}}. You will be prompted to input an API key to configure authentication with an AI provider. This key will not be exposed outside of the host machine.

```sh
curl -Ls https://get.konghq.com/ai | bash
```

> Follow the prompts to set up the AI Proxy plugin with the LLM provider of your choice. 

Reviewer: For some reason the suggestion thing isnt working but I would delete this.
